### ISSUE TITLE
Evaluate speed when adding empty additional columns

### ISSUE THREAD

**Issue by lkiesow** [emissions-api/emissions-api#151]
Check what happens if an empty new column is added and if this makes things slower. What happens if these are filled?

**Comment by shaardie** [comment]
**TL;DR:** There is no measurable effect on the queries when adding a new column. no matter if it is filled with values or not.

I used `\timing` to measure the time, a database with a month of carbon monoxide data and the following query

```sql
SELECT AVG(value) from carbonmonoxide LIMIT 10000;
```

For each test I executed the query 100 times. To minimize the affect of irregular measurements I removed the 10 highest and 10 lowest results and calculated the average time from the remaining ones.

First I tested this with the regular database and got 1.557s.

After that I added a new column with

```sql
ALTER TABLE carbonmonoxide ADD value2 double precision;
```

and ran the test again with a result of 1.571s.

And at last I added a value to the new column by

```sql
UPDATE carbonmonoxide SET value2 = 0.1235132416;
VACUUM FULL;
```

The [`VACUUM`](https://www.postgresql.org/docs/current/sql-vacuum.html) is important here, since `UPDATE` duplicates the rows and not delete the old ones. This is messing up the database. The result was 1.497s.

So the maximal deviation is less than 4%. And that even in the wrong direction, since the time decrease. I would call that noise.

@lkiesow: Please close this issue if you are happy with the result.
